exampleCourse/questions/demo/custom/gradeFunction/server.py

```diff
@@ -8,6 +8,23 @@ def generate(data):
     data["correct_answers"]["c"] = c
 
 
+def parse(data):
+    sub = data["submitted_answers"].get("c", "")
+    if len(sub.replace("0", "").replace("1", "")) > 0:
+        # This question uses both feedback and format_errors for reference, but
+        # only one is needed.
```

Comment:
I feel like the idiomatic solution here is to use `format_errors` for the 0/1 check? In that case, I'd prefer to write this example only using `format_errors` and not use `feedback` at all, so that this is a valid example of how we'd really write the code. Perhaps we could have a second custom grader example question to show the use of `feedback`?

Reply:
Ah, I see you updated the example! Nice, I like the new one.

Reply:
1641306dd has two parsing tests, one using format errors (invalid digits), one using feedback (leading zeros).